base/src/main/java/com/smartdevicelink/protocol/SdlProtocolBase.java

```diff
@@ -84,7 +84,7 @@
     public static final int V2_HEADER_SIZE = 12;
 
     //If increasing MAX PROTOCOL VERSION major version, make sure to alter it in SdlPsm
-    private static final Version MAX_PROTOCOL_VERSION = new Version("5.1.0");
+    private static final Version MAX_PROTOCOL_VERSION = new Version("5.2.0");
```

Comment:
As long as we're altering this line, please change to the other constructor `new Version(5, 2, 0);`